### PR TITLE
Replace mockredis with fakeredis

### PR DESCRIPTION
**Pull request by kleesc**
### Description of Changes

* [PROJQUAY-134](https://issues.redhat.com/browse/PROJQUAY-134)

#### Changes:

* Replaces mockredispy (deprecated) with fakeredis
* ..

#### Issue: <link to story or task>

**TESTING** ->

**BREAKING CHANGE** ->

---

## Reviewer Checklist

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -93,8 +94,6 @@ MarkupSafe==1.1.1
 maxminddb==1.4.1
 meld3==2.0.0
 mixpanel==4.5.0
-mock==3.0.5
```
Comment (thomasmckay) [inline requirements.txt]: Keep mock as it is used in a number of tests.

Comment (kleesc) [reply requirements.txt]: It should be included in the standard lib
